broker/lua/src/macro_cache.cc

```diff
@@ -907,6 +946,38 @@ void macro_cache::_process_custom_variable(
   }
 }
 
+/**
+ *  Process a custom variable event.
+ *  The goal is to keep in cache only custom variables concerning severity on
+ *  hosts and services.
+ *
+ *  @param data  The event.
+ */
+void macro_cache::_process_pb_custom_variable(
+    std::shared_ptr<io::data> const& data) {
+  neb::pb_custom_variable::shared_ptr cv =
+      std::static_pointer_cast<neb::pb_custom_variable>(data);
+  if (cv->obj().name() == "CRITICALITY_LEVEL") {
+    int32_t value;
+    if (!absl::SimpleAtoi(cv->obj().value(), &value)) {
+      SPDLOG_LOGGER_DEBUG(log_v2::lua(),
+                          "lua: processing custom variable representing a "
+                          "criticality level for "
+                          "host_id {} and service_id {} and level {}",
+                          cv->obj().host_id(), cv->obj().service_id(), value);
+      if (value)
```

Comment:
I think you made a little mistake here. When SimpleAtoi failed it returns False. So you log a debug message when there is an error and this message does not talk about an error... Weird, isn't it?